README.md

```diff
@@ -166,6 +166,29 @@ def lambda_handler(event, context):
     }
 ```
 
+## Replacing the SSM client
+
+You can replace the SSM client for testing by setting the `SSMParameterStore.SESSION_FACTORY` and / or the `SSMParameterStore.CLIENT_FACTORY` class variables.
+
+For example, if you are using the `placebo` library to mock out `boto3` behaviour:
+
+```python
+import boto3
+import placebo
+
+from ssm_cache.cache import SSMParameterStore
```

Comment:
You can simply `from ssm_cache import SSMParameterStore` by exposing `SSMParameterStore` in the `__init__.py` file :)

Reply:
👍

Reply:
I remember now why I didn't do that originally -- `pylint` complained about line length, and I justified leaving it out as "this is something the default use case won't need to access".

I've split into 2 `import` lines in `__init__.py` to address the line length.

Reply:
Alternatively, this is totally ok as well:

```python
from your_module import (
    something1,
    something2,
    ...
)
```